scripts/segmentation_template.md

```diff
@@ -205,10 +201,21 @@ print(f"Segment time: {seg[0].start} to {seg[0].stop}")
 print(f"Dropsonde launch times: {ds_drops.time.sel(time=seg_drops).values}")
 ```
 
+## Identify visually which straight_leg segments lie on EC track
+
+```python
+seg = sl1
+plt.plot(ds.lon.sel(time=slice(takeoff, landing)), ds.lat.sel(time=slice(takeoff, landing)))
+plt.plot(ds.lon.sel(time=seg[0]), ds.lat.sel(time=seg[0]), color = 'red')
+plt.scatter(ds_drops.lon, ds_drops.lat, s=10, c="k")
+plt.xlabel("longitude / °")
+plt.ylabel("latitude / °");
+```
+
```

Comment:
From the heading I would have expected to see a plot that loads the EC track (forecast) data and plots it maybe with an accepted uncertainty range of few hundret kilometer left and right and displays this track with overlapping straight legs. But that might be out of scope as we usually know the EC track location from the flight plans/reports. The current code block seems very redundant to the one above and I'd rather suggest to not include it and leave it up to the user to basically add the following line to the plot before.
```python
plt.plot(ds.lon.sel(time=slice(takeoff, landing)), ds.lat.sel(time=slice(takeoff, landing)))
```